tests/secc/states/test_messages.py

```diff
@@ -118,11 +118,15 @@ def get_dummy_v2g_message_session_stop_req():
     )
 
 
-def get_dummy_v2g_message_authorization_req():
+def get_dummy_v2g_message_authorization_req(
+    id: Optional[str] = None, gen_challenge: Optional[bytes] = None
+):
     # The AuthorizationReq is empty, unless it is following a PaymentDetailsRes
     # message, in which case it must send back the generated challenge.
-    authorization_req = AuthorizationReq()
-
+    if gen_challenge:
+        authorization_req = AuthorizationReq(id=id, gen_challenge=gen_challenge)
+    else:
+        authorization_req = AuthorizationReq()
```

Comment:
if you are not generating here the challenge and just injecting, there is no point on having the if clause.
I would just pass along the injection, but is a minor thing